src/components/filters/cost-filter.tsx

```diff
@@ -120,6 +129,13 @@ export function CostFilter({ id }: { id: number }) {
           label={<i className="icon-x" />}
           onCheckedChange={onSetX}
         />
+        <Checkbox
```

Comment:
We could add `cols={2}` to the `<CheckboxGroup />` and re-order the  checkboxes like this? What do you think?

![image](https://github.com/user-attachments/assets/fd448295-5774-4b1b-ac2b-822dd1069c0c)

Reply:
hell yeah